pkg/reconciler/testing/broker.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"context"
+	"time"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// BrokerOption enables further configuration of a Broker.
+type BrokerOption func(*v1alpha1.Broker)
+
+// NewBroker creates a Broker with BrokerOptions.
+func NewBroker(name, namespace string, o ...BrokerOption) *v1alpha1.Broker {
+	b := &v1alpha1.Broker{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "eventing.knative.dev/v1alpha1",
+			Kind:       "Broker",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+		},
+		Spec: v1alpha1.BrokerSpec{
+			ChannelTemplate: &v1alpha1.ChannelSpec{},
```

Comment:
I would probably leave this unspecified.

Reply:
Done